pkg/reconciler/kubernetes/tektoninstallerset/install.go

```diff
@@ -320,6 +326,24 @@ func (i *installer) AllDeploymentsReady() error {
 	return nil
 }
 
+func (i *installer) IsJobCompleted() error {
+	for _, u := range i.Manifest.Filter(jobPred).Resources() {
+		resource, err := i.Manifest.Client.Get(&u)
+		if err != nil {
+			return err
+		}
+		job := &batchv1.Job{}
+		if err := scheme.Scheme.Convert(resource, job, nil); err != nil {
+			return err
+		}
+		if !isJobCompleted(job) {
+			return fmt.Errorf("Job not successful")
+		}
+	}
+
+	return nil
+}
+
```

Comment:
From installerSet stand point isJobRunninig is something that "might" come under the responsibility of installerset. The success and failure of a jobs is upto the parent reconciler to decode and interpret. 

If if makes sense, from TektonHub reconciler, create an additional installerset specifically for this job.

However, i don't have a strong opinion here. If it makes more sense to you to add this generic job completion check to installerSet reconciler, then we can keep this here.